### PR TITLE
chore(old-edit-content): Update Binary Field on save

### LINKED ISSUE #28337: Binary Field: Information of assets needs be refreshed after any change
Related to #28115

We are not refreshing the asset information after you do some change and publish it, you need to close the modal and reopen to see the changes. 
Attached a video explaining the problem: 

https://github.com/dotCMS/core/assets/2641437/e4f1d21b-d971-424c-88c9-d07b342990f6

Basically we need to update the generating information every time you click on this button.

_Originally posted by @bryanboza in https://github.com/dotCMS/core/issues/28115#issuecomment-2073039999_

### PR DESCRIPTION
**Pull request by zJaaal**
### Proposed Changes
* Reuse init function of Binary Field on JSP to set Inputs on save

### Screenshots

https://github.com/user-attachments/assets/5381c380-0e83-4a76-9dce-c1621d9481e4

This PR fixes: #28337

### CODE REVIEW

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp [hunk 1]
```diff
@@ -62,6 +62,8 @@
     Object inodeObj =(Object) request.getAttribute("inode");
     String inode = inodeObj != null ? inodeObj.toString() : "";
 
+    String contentletIdentifierGlobal = contentlet.getIdentifier();
```
Comment (nollymar) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:65]: This jsp uses the contentletIdentifier in multiple places. I would pull the `String contentletIdentifier = contentlet.getIdentifier();` out the Story Block segment (line 215) and declare it global, or modified that line to use your declared variable. I simply have one global with the contentlet identifier

Comment (zJaaal) [reply dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:65]: Good catch

Comment (zJaaal) [reply dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:65]: Done!

**Review by fmontes — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp [hunk 2]
```diff
@@ -767,22 +768,29 @@
...
-                
+
                 function closeConfirmReplaceName(){
                     dijit.byId("confirmReplaceNameDialog-<%=field.getVelocityVarName()%>").hide();
                 }
             </script>
             <script>
                 // Create a new scope so that variables defined here can have the same name without being overwritten.
-                (function autoexecute() {
+                function setBinaryFieldInputs() {
```
Comment (rjvelazco) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:778]: What happens if you have more than one Binary field in the same `content type`?

**Review by rjvelazco — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp (hunk 2 above)
Comment (zJaaal) [reply dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:778]: I will check this. I think it will fail

Comment (zJaaal) [reply dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:778]: It did fail, but I already got the fix. Outstanding catch

**Review by rjvelazco — APPROVED** [review]

**Pull request merged**